### PR TITLE
Prepare for CI/CD

### PR DESCRIPTION
**Pull request by pingzing**
This sets up the Azure Pipelines to build and prepare artifacts for deployment to the test server. It also makes a few important parts of the server more environment-agnostic.

### CODE REVIEW

File: src/main.ts [hunk 1]
```diff
@@ -1,10 +1,11 @@
 import { NestFactory } from '@nestjs/core';
 import { config, DotenvConfigOutput } from 'dotenv';
 import * as cookieParser from 'cookie-parser';
+import * as path from 'path';
 
 import { AppModule } from './app.module';
 
-const results: DotenvConfigOutput = config();
+const results: DotenvConfigOutput = config({path: path.resolve(__dirname, '..', '.env')});
```
Comment (pingzing) [inline src/main.ts:8]: Or, in plain English: look in whatever directory is one higher than `main.js` for the `.env` file.

**Review by Figments — APPROVED** [review]
Everything looks good! Let's merge it in.
